### ISSUE TITLE
German translation

### ISSUE THREAD

**Issue by WBTMagnum** [w3c/wcag-em-report-tool#338]
It would be nice to have a German translation of the tool itself (for clarity: that does not include the WCAG translations and must be covered separately).

We already startet with the translation of the tool and have a first draft:
 * [Our branch for the German translation](https://github.com/WIENFLUSS/wcag-em-report-tool/tree/german-translation)
 * We also have a test instance (for review): https://wcag-em.tools.wienfluss.net

What would be the procedure to approve the translations and bring them in?

**Comment by rvantonisse** [comment]
Hi @WBTMagnum , sorry for the late response!

> What would be the procedure to approve the translations and bring them in?

You can always start with an initial pull request of your translation. To review the translation it is best if there are any other native speaking Germans that can help.
I can review the code for missing translations, other formatting and a quick comparison with a generated German translation.

@nitedog do you know if there is a procedure, best practice or requirement for approval of translations here?
